.travis.yml

```diff
@@ -10,14 +10,21 @@ os:
   - osx
 
 script: ./bin/travis.sh
-env: TASK=test
+env: TASK=test COVERAGE=false
 install: npm install --ignore-scripts
 
 after_success:
-  - npm run coverage:ci
+  - if [ "${TRAVIS_OS_VERSION}" == "linux" && "${TRAVIS_NODE_VERSION}" | cut -d'.' -f1 == "10"]; then npm run coverage:ci; fi
 
 matrix:
+  exclude:
```

Comment:
What does this column do?

Reply:
It excludes the jobs matching the criteria from CI. 

https://docs.travis-ci.com/user/customizing-the-build#Excluding-Jobs